rom/laghos_rom.hpp

```diff
@@ -38,6 +39,19 @@ enum offsetStyle
     interpolateOffset
 };
 
+enum samplingType
+{
+    randomSampling,
+    latinHypercubeSampling
+};
+
+enum errorIndicatorType
+{
+    useLastLiftedSolution,
+    varyTimeStep,
+    varyBasisSize
+};
```

Comment:
Are there only tests for `useLastLiftedSolution`? The `vary*` options are not documented in `GREEDY.md`, and I'm not sure if they are really supported. For example, I don't see where `dt_factor` is ever changed from 1.

Reply:
If this seems a bit backwards, I can instead add another greedyDt option

Reply:
Ah never mind. Just noticed something wrong about it. Modifying to have a greedyDt option.